server/src/main/java/org/eclipse/openvsx/adapter/VSCodeAdapter.java

```diff
@@ -383,6 +387,9 @@ public ModelAndView download(@PathVariable String namespace, @PathVariable Strin
                     .collect(Collectors.joining(","));
             queryVer.addProperty(PROP_EXTENSION_PACK, bundledExtensions);
             queryVer.addProperty(PROP_LOCALIZED_LANGUAGES, "");
+            if (isWebExtension(extVer)) {
+                queryVer.addProperty(PROP_WEB_EXTENSION, "true");
+            }
```

Comment:
The query output stays backward compatible by adding the WebExtension prop only in the presence of a web extension.